CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Fixed
```

Comment:
This should be `### Added` as you are adding support for graylog ` >= 2.5` I would not consider this a bug but rather an enhancement as it adds support for a version that was not out when this check was created. Yes it's fairly nuanced and I can fix that up for you post acceptance prior to release if you want.